test/BuildTools.Tasks.Tests/UnzipArchiveTest.cs

```diff
@@ -54,6 +54,30 @@ public void UnzipsFile()
             }
         }
 
+        [Fact]
+        public void Overwrites()
+        {
+            var files = new[]
+            {
+                "a.txt",
+                "dir/b.txt"
+            };
+
+            var dest =CreateZip(files);
+            var outDir = Path.Combine(_tempDir, "out");
+
+            var task = new UnzipArchive
+            {
+                File = dest,
+                Destination = outDir,
+                BuildEngine = new MockEngine(),
+                Overwrite = true
+            };
+
+            Assert.True(task.Execute(), "Task failed");
```

Comment:
This is the message that appears when the assertion fails. A better message might be "The task failed but should have passed".